### PR TITLE
rfkill: add page

### PR DESCRIPTION
**Pull request by BinaryBrain**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/rfkill.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+`rfkill block {{bluetooth}}`
+
+- Unblock by type (e.g. bluetooth, wlan):
+
+`rfkill unblock {{wlan}}`
+
+- Filter by columns:
+
+`rfkill -o {{COL1,COL2,COL3}}`
```
Comment (sbrl) [inline pages/linux/rfkill.md]: Perhaps a concrete example would be better here?

File: pages/linux/rfkill.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# rfkill
+
+> Enable and disable wireless devices.
+
+- List devices:
+
+`rfkill list {{all}}`
```
Comment (agnivade) [inline pages/linux/rfkill.md]: Should all be in tokens here ? What other values are allowed ?

File: pages/linux/rfkill.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Block by type (e.g. bluetooth, wlan):
```
Comment (agnivade) [inline pages/linux/rfkill.md]: "Block _device_ by type .. " just to be slightly clearer

File: pages/linux/rfkill.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Unblock by type (e.g. bluetooth, wlan):
+
+`rfkill unblock {{wlan}}`
+
+- Filter by columns:
+
+`rfkill -o {{COL1,COL2,COL3}}`
+
+- Output JSON:
```
Comment (agnivade) [inline pages/linux/rfkill.md]: "Output in JSON format"

**Comment by agnivade** [comment]
@BinaryBrain - Kindly take care of the travis failure. :)

**Pull request merged**